grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -211,12 +211,17 @@ class StockMovementController {
                 try {
                     stockMovementService.deleteStockMovement(params.id)
                     flash.message = "Successfully deleted stock movement with ID ${params.id}"
+                    params.deleted = 'true'
```

Comment:
I'd add a comment above why it is needed and also can't it be boolean here instead of the string?

Reply:
I did this accidentally, in other places i use boolean instead of string.